backend/src/handlers/billHandler.ts

```diff
@@ -78,7 +78,7 @@ async function DeleteBill(billid: String): Promise<void> {
 }
 
 //Gets single bill by billid
-async function GetBill(billid: string): Promise<Bill> { 
+async function GetBill(billid: String): Promise<Bill> { 
     const client = await Pool.connect();
     try {
         const result = await client.query('SELECT * FROM bill WHERE billid = $1', [billid]);
```

Comment:
What is this for?

Reply:
This is for Allocation, to get its parent